massa-models/src/secure_share.rs

```
@@ -81,6 +78,19 @@ where
         })
     }
 
+    /// Compute hash
+    #[allow(unused_variables)]
+    fn compute_hash(
+        content: &Self, // use only for Denounce able object
+        content_serialized: &[u8],
+        content_creator_pub_key: &PublicKey,
+    ) -> Result<Hash, SerializeError> {
```

for simplicity, this does not need to return an error

---

I need to return this kind of error because I use a DenunciationDataSerializer (for Endorsement & Block header)

---

now the Result can be removed

---

Removed